### DISCUSSION TITLE
Harmony is Amazing

### DISCUSSION THREAD

**Discussion by TonyValenti** [pardeike/Harmony#532] [Show and tell]
Hi @pardeike -
I just wanted you to know that harmony is amazing.  I'm using it in a really neat way to instrument a 3rd party app that isn't behaving the way it should. 

1.  I use a .net startup hook to inject my code into the host process at application start. 

2.  I use harmony to override the an HttpMessageHandler constructor so that I can inject a custom HttpClientHandler into all HttpClients. 

3.  My custom handler intercepts/rewrites http requests. 

Anyway, I thought the above was super cool and just wanted you to know you enable some really awesome solutions outside of gaming.

**Comment by pardeike** [comment]
Thank you!!
